src/App/components/Tasks.js

```diff
@@ -0,0 +1,74 @@
+import React, { memo, useContext } from 'react';
+import styled from 'styled-components';
+import { TaskContext } from '../contexts/taskContext';
+import { DeleteButton } from './DeleteBtn';
+import RoundCheckBox from './Checkbox';
+
+const ListItem = styled.li`
+  position: relative;
+  font-size: 24px;
+  border-bottom: 1px solid #ededed;
+
+  &:hover .DeleteBtn {
+      display: block;
+      cursor: pointer;
+  };
```

Comment:
What's `.DeleteBtn`?

Reply:
This is stand for the delete button, I renamed it to make it more clear

Reply:
Please do not code this way.

Reply:
About this, can I have a short call with you tomorrow? I'm listing all the problem I'm stuck with